Prepare for Debugbar 3.x

barryvdh opened the pull request:
- inline svg icons
 - remove deprecated jquery bind
 - copy build/lint scripts

barryvdh commented:
Hmm the typehints made it a bit more then anticipated, but overall a good change. Just a lot of work. But basics seem to be working again now,.

erikn69 commented:
Before
<img width="219" height="144" alt="image" src="https://github.com/user-attachments/assets/ea0ec639-4c0b-4d09-8040-e1a29e43b7eb" />
After
<img width="190" height="71" alt="image" src="https://github.com/user-attachments/assets/666e0ded-aa2c-4541-af9f-bc80cf4f0731" />

on GateCollector, expand button(_sf-dump args_) is missing, is it an expected behavior?

erikn69 commented on src/DataCollector/GateCollector.php:
```diff
@@ -17,14 +17,9 @@
 class GateCollector extends MessagesCollector
 {
-    /** @var int */
-    protected $backtraceLimit = 15;
```
After this change, link editor always shows GateCollector.php
Before
<img width="262" height="78" alt="image" src="https://github.com/user-attachments/assets/26a16131-07d7-43d8-829e-b7b890b9ee95" />
After
<img width="256" height="66" alt="image" src="https://github.com/user-attachments/assets/0b6be039-1972-471c-bfd7-bae6c7f1fc19" />

barryvdh commented:
> Before <img alt="image" width="219" height="144" src="https://private-user-images.githubusercontent.com/4933954/530387814-ea0ec639-4c0b-4d09-8040-e1a29e43b7eb.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjY3NzkzODksIm5iZiI6MTc2Njc3OTA4OSwicGF0aCI6Ii80OTMzOTU0LzUzMDM4NzgxNC1lYTBlYzYzOS00YzBiLTRkMDktODA0MC1lMWEyOWU0M2I3ZWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTIyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTEyMjZUMTk1ODA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWFlYTgxMGIyMzllNjMyNDBjNTllNmEzNTZiMTBlZTEyNzM1YmRjYmMxMzQ5M2E1MmNkZjdlODU2ZmQxYThlNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.2VPR5588v_iGIoBctQtt5KGcn-oK_c0O8MfL8ZGY9nU"> After <img alt="image" width="190" height="71" src="https://private-user-images.githubusercontent.com/4933954/530387980-666e0ded-aa2c-4541-af9f-bc80cf4f0731.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjY3NzkzODksIm5iZiI6MTc2Njc3OTA4OSwicGF0aCI6Ii80OTMzOTU0LzUzMDM4Nzk4MC02NjZlMGRlZC1hYTJjLTQ1NDEtYWY5Zi1iYzgwY2Y0ZjA3MzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTIyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTEyMjZUMTk1ODA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTg2ZTI3ZWFiOGMwMjhhYWRiMWE4ZGQ3MTdmMTRiMDFkOTU0Nzg1ZGI5YmMzZGE4ZGQ5MWEyMTRiNDMyMjU0YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.h5Q6XA3nuAhXM9MW9NjehVTM-n2rSup9zMkIIXqcJRE">
> 
> on GateCollector, expand button(_sf-dump args_) is missing, is it an expected behavior?

Hmm not sure why this is happening. I see the dump header getting outputted. It's working on plain php-debugbar, and a normal `dump` is also still correct..

barryvdh commented:
This is fixed. jQuery was executing the scripts that were loaded in the messages, the plain JS version does not.
Fixed in https://github.com/php-debugbar/php-debugbar/pull/841
Tagged a new beta with the fixes for symfony dump, options panel and backtrace.
Also added the PR for the global settings and removed the override here.

parallels999 commented:
a small css problem on timeline tab
fix `overflow-x: hidden;`

<img width="262" height="220" alt="image" src="https://github.com/user-attachments/assets/47e44bb1-ed08-46cf-8c77-37386e260b16" />

barryvdh commented:
> In QueriesCollector, I noticed that the editor links used to be at the end, now they're at the beginning. Was this expected?
> 
> <img alt="image" width="339" height="41" src="https://private-user-images.githubusercontent.com/109294935/534787587-354eeb7f-a1e3-47e8-bc8b-ad687c19e09a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjgyNTU1MzAsIm5iZiI6MTc2ODI1NTIzMCwicGF0aCI6Ii8xMDkyOTQ5MzUvNTM0Nzg3NTg3LTM1NGVlYjdmLWExZTMtNDdlOC1iYzhiLWFkNjg3YzE5ZTA5YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDExMlQyMjAwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZGJiNjFjYWI1NmUxYjllZDVlMzE2NDQ3MWZiMGVkYzVjNzU3MTZiMTExZDU1MDQ3MTQxNzQ1OGUyNTFmMDBlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.llAGAtCZNxSzOSXc9g_xZjh5Xt4pljbZwRYBioH6JHc">

No not expected

barryvdh commented:
Fixed in https://github.com/php-debugbar/php-debugbar/pull/947
